postgres-appliance/Dockerfile

```diff
@@ -48,7 +48,7 @@ ARG TIMESCALEDB
 ARG TIMESCALEDB_APACHE_ONLY=true
 ARG TIMESCALEDB_TOOLKIT=true
 ARG COMPRESS
-ARG PGOLDVERSIONS="12 13 14 15"
+ARG PGOLDVERSIONS="13 14 15"
```

Comment:
What is the harm in leaving this equal to upstream? I'm thinking that it's better to align as much as possible.

Reply:
Yes, let's go with as much upstream as possible

Reply:
Dropping that commit.